plugins/destination/bigquery/Makefile

```diff
@@ -0,0 +1,23 @@
+PACKAGE_NAME          := github.com/cloudquery/cloudquery/plugins/destination/bigquery
+
+.PHONY: test
+test:
+# we clean the cache to avoid scenarios when we change something in the db and we want to retest without noticing nothing run
+	go clean -testcache
+	go test -timeout 3m ./...
+
+.PHONY: lint
+lint:
+	@golangci-lint run --config ../../.golangci.yml --timeout 10m --verbose
+
+.PHONY: release-dry-run
+release-dry-run:
```

Comment:
THat's a copy/paste from snowflake. There is no need for `release-dry-run` for BigQuery given it just work (no CGO stuff)